etna/ensembles/direct_ensemble.py

```diff
@@ -0,0 +1,134 @@
+from copy import deepcopy
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+
+import numpy as np
+from joblib import Parallel
+from joblib import delayed
+
+from etna.datasets import TSDataset
+from etna.ensembles import EnsembleMixin
+from etna.pipeline.base import BasePipeline
+
+
+class DirectEnsemble(BasePipeline, EnsembleMixin):
+    """DirectEnsemble is a pipeline that forecast future values merging the forecasts of base pipelies.
```

Comment:
> DirectEnsemble is a pipeline that forecast future values merging the forecasts of base pipelies

DirectEnsemble is a pipeline that forecast[s] future values merging the forecasts of base pipeli[n]es